dynawo/sources/Common/Dictionaries/DYNLog_en_GB.dic

```diff
@@ -181,6 +181,7 @@ SolverVarDisplayLimit         =             display limited to %1% variables
 SolverNbZVar                  =             number of discrete variables : %1%
 // --> DYNSolverSIM
 SolverSIMInitOK               =             initialization of SIM solver : ok
+SolverSIMInitGuessOK          =             initial guess of KINSOL is accurate, following solver iterations will be skipped until the next mode change
```

Comment:
I would make it more generic if we use another solver later on: initial guess of the algebraic solver is accurate

Reply:
Solved